public/docs/advanced/commands.md

```diff
@@ -132,55 +218,94 @@ Optional parameters:
+Takes no arguments. All parameters are pulled from your project's configuration file. See the [Package Management with EthPM](/docs/getting_started/packages-ethpm) section for more details.
+
 
 ### serve
 
-Serve the built app from `http://127.0.0.1:8080`, rebuilding and redeploying changes as needed. Like `truffle watch`, but with the web server component added.
+Serve the built app from `http://127.0.0.1:8080`, rebuilding and redeploying changes as needed. Similar to `truffle watch`, but with the web server component added.
 
-```none
-$ truffle serve
+```shell
+truffle serve [-p <port>] [--network <name>]
 ```
 
 Optional parameters:
 
-* `-p port`: Specify the port to serve on. Default is 8080.
-* `--network name`: Specify the network to use, using artifacts specific to that network.
+* `-p <port>`: Specify the port to serve on. Default is 8080.
+* `--network <name>`: Specify the network to use, using artifacts specific to that network. Network name must exist in the configuration.
+
 
 ### test
 
-Run all tests within the `./test` directory, or optionally run a single test.
+Run Mocha and Solidity tests.
 
-```none
-$ truffle test [/path/to/test/file]
+```shell
+truffle test <test_file> [--compile-all] [--network <name>] [--verbose-rpc]
 ```
 
+Will run some or all tests within the `test/` directory, as specified.
+
+Parameters:
+
+* `<test_file>`: Name of the test file to be run. Can include path information if the file does not exist in the current directory.
+
 Optional parameters:
 
-* `--compile-all`: Compile all contracts instead of intelligently choosing.
+* `--compile-all`: Compile all contracts instead of intelligently choosing which contracts need to be compiled.
+* `--network <name>`: Specify the network to use, using artifacts specific to that network. Network name must exist in the configuration.
 * `--verbose-rpc`: Log communication between Truffle and the RPC.
-* `--network name`: Specify the network to use, using artifacts specific to that network.
+
+### unbox
+
+Unbox Truffle project.
+
+```shell
+truffle unbox box_name
+```
+
+Download a [Truffle Box](/boxes) to the current working directory. See the [list of available boxes](/boxes).
+
+Parameters:
+
+* `box_name`: Name of the Truffle Box.
+
 
 ### version
 
 Show version number and exit.
 
-```none
-$ truffle version
+```shell
+truffle version
 ```
 
 ### watch
 
-Watch for changes to contracts, app and configuration files. When there's a change, rebuild the app if necessary.
+Watch filesystem for changes and rebuild the project automatically.
 
-```none
-$ truffle watch
+```shell
+truffle watch
 ```
+
+This command will initiate a watch for changes to contracts, application, and configuration files. When there's a change, it will rebuild the app as necessary. Similar to `truffle serve`, but without the web server component.
```

Comment:
This is another command that should be deprecated. For this one, something like:

> This command is deprecated. Please use external tools to watch for filesystem changes and rerun tests.

Reply:
Got it.

Should we be notating these deprecations somewhere else? Perhaps in the command description in `truffle help` or as a console note during their execution?